templates/globals.py

```diff
@@ -0,0 +1 @@
+TIMEOUT_DEPLOY = 7200  # 120 minutes
```

Comment:
This file will make the zrobot crash, remove it and import the variable from jumpscale_lib